### DISCUSSION TITLE
Winmerge parent folder navigation is broken

### DISCUSSION THREAD

**Discussion by IgorT75** [WinMerge/winmerge#2568] [General]
After certain version Winmerge is not accepting Backspace to navigate to parent folder.

I.e. if I compare folder D:\StreamOne\DirA\DirB with D:\StreamTwo\DirA\DirB I see files compared in DirB folders.
Old behavior was that hitting Backspace changes compared folders to DirA.
But this does not work anymore. Tree mode does not affect it.

Is it intentional change and is there a way to get back to old behavior?

**Comment by sdottaka** [comment]
In both the current and older versions, pressing the backspace key will move to the parent folder only if the "Include subfolders" option is disabled.

In version 2.16, the "Include subfolders" option is enabled by default.

The Include subfolders option can be disabled in three ways:

[Compare/Folder category in the Options dialog]
![image](https://github.com/user-attachments/assets/f83f20ab-4d89-4202-a123-5ab92eac2541)

[Arrow to the right of the Options button on the toolbar]
![image](https://github.com/user-attachments/assets/7d4c5435-7a45-4e14-9a5a-feeb41d167d6)

[Select files or folders window]
![image](https://github.com/user-attachments/assets/8042f1fd-05cf-473d-a43f-cd3982272ac0)
